cmd/server/assets/header.html

```diff
@@ -103,10 +103,12 @@ <h6 class="dropdown-header">Manage realm</h6>
       {{end}}
 
       {{if .currentUser.SystemAdmin}}
+      {{if not (.currentPath.IsDir "/admin")}}
```

Comment:
I don't see any reason to remove this - we don't do it w/ other pages

Reply:
maybe "Launch" just feels funny to me when you're already there

Reply:
what if we link to specific pages here instead?

![image](https://user-images.githubusercontent.com/92319/98307795-80f65d80-1f7b-11eb-8acf-d706521a7416.png)

Reply:
How about we just highlight the active nav item (see most recent commit)